Makefile

```diff
@@ -151,8 +151,16 @@ performance_bench_report: is_docker_running
 dev_stack_run: server
 	HASH=$(hash) docker-compose -f containers/docker-compose-integration-tests.yml --project-name=integration-tests-$(hash) down
 	HASH=$(hash) docker-compose -f containers/docker-compose-dev-stack.yml --project-name=dev-stack-$(hash) down
+	HASH=$(hash) docker-compose -f containers/docker-compose-semi-dev-stack.yml --project-name=dev-stack-$(hash) down
 	HASH=$(hash) docker-compose -f containers/docker-compose-dev-stack.yml --project-name=dev-stack-$(hash) up
 
+.PHONY: semi_dev_stack_run
```

Comment:
there are ways to tell a docker-compose file to not start a service, or to start a particular image defined in the command. Investigate ways to remove this redundant (and hard to maintain docker-compose file). I think in the future we're going to consolidate all the compose files.